### ISSUE TITLE
get file in use?

### ISSUE THREAD

**Issue by 56curious** [jbardin/scp.py#137]
Hello

Requiring to get a file that is in use by a server file that contains logs etc. but permission denied errror 13 is returned. Saw a post: https://stackoverflow.com/questions/53760297/python-paramiko-permissionerror-errno-13-permission-denied-when-get-files-fr saying that you can't get a fil that is in use. is there a work around for this or is it not the case? Can provide code if needed.

**Comment by remram44** [comment]
This StackOverflow question has unrelated coding errors. Does your code exhibit the same errors, or are you confident the problem is actually that the file is in use?

**Comment by 56curious** [comment]
Hi yes I ignored the code errors but saw a response about the file being in use and the error occouring.
So the remote location constantly saves to a file called arma3server-console.log which is in use by the arma3server.exe and when the game server shuts down it writes it's date, time etc. to it and when the server boots up again it will make this temp file again. 
The code I use to get this file is:
```
client = paramiko.SSHClient()
client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
client.connect(hostname=sftp_hostname,port=sftp_port,username=sftp_username,password=sftp_password)
ftp_client=client.open_sftp()
ftp_client.get("/operations/log/arma3server-console.log", sftp_source)
await ctx.message.channel.send("The file was downloaded successfully!")
ftp_client.close()
```
where sftp_source is just `os.getcwd()` to get a directory. 

I just tried to download one of these date-time files again and it gave me the same error, something i'm sure I tested before and that had worked (although was late at night haha)
It's possible my code is wrong or i've missed something obvious here ha, I am an amateur at this :D.
